### PR TITLE
Refactor makefile and nexus init

### PR DESCRIPTION
**Pull request by akaRem**

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -32,7 +38,7 @@ test: fmtcheck
 		xargs -t -n4 go test -cover $(TESTARGS) -timeout=30s -parallel=4
 
 testacc: fmtcheck
-	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -cover -timeout 120m -parallel=4
```
Comment (kuzmacska) [inline GNUmakefile]: why did you remove the var?

Comment (akaRem) [reply GNUmakefile]: By accident, fixed

**Review by kuzmacska — APPROVED** [review]
